### PR TITLE
Add Android Platform support

### PR DESCRIPTION
**Pull request by aicodex**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -51,17 +51,11 @@ if(ANDROID)
   #   mips, mips64
   # }
   # TODO: Support more than just "arm" in LibUtils
```
Comment (gpu) [inline CMakeLists.txt]: This TODO only refers to Android, right?

**Comment by gpu** [comment]
In doubt, I'd try to allocate some time (possibly next week) to apply the changes to the code and makefile locally, and create a ZIP from the resulting JAR, together with the native libraries, so that you can try out whether the result works on your system.

**Comment by aicodex** [comment]
Oh I'd like to be the first person to try tomato. Ha Ha.

**Comment by gpu** [comment]
Well, it will still contain *your* binaries and your changes, so it's *your* tomato ;-) I'd just like to avoid having the natives in the repo, and wanted to cover the `"arm"` case - that's really a point where changes are difficult to test (due to the many different `arch` types out there)

**Comment by gpu** [comment]
I assume that JOCL essentially "works" on Android, as of https://github.com/gpu/JOCL/pull/44 , and going to close this one.

**Pull request closed without merging**